CMakeLists.txt

```diff
@@ -581,6 +581,9 @@ if (BUILD_TESTING)
             -Wshadow  -Wcast-align -Wwrite-strings -Wformat-security
             -Wno-deprecated-declarations -Wno-unknown-pragmas -Wno-deprecated -std=gnu99
         )
```

Comment:
You're going to have to add -Wno-missing-braces to this list on line 582. I don't see any other warnings that need to be suppressed in the build output, so maybe this is the last fix?